packages/core/src/index.ts

```diff
@@ -113,10 +142,7 @@ export const create = (config: Config): VoiceCompass => {
       if (isDomElement(node)) {
         const vcAttributes = readVcAttributes(node, "click");
         if (vcAttributes) {
-          updateStep({
-            ...vcAttributes,
-            journeyId: vcAttributes.journeyId || config.journeyId,
-          });
+          updateStep(vcAttributes);
```

Comment:
This seems logically correct since `updateStep` is already looking at `config.journeyId`, but I'm not comfortable approving while it is in the same PR as other things. Please revert and pull out into a PR that only does this and nothing else.

Reply:
yup, ill separate this to another PR